subsys/debug/Kconfig

```diff
@@ -382,6 +382,7 @@ config EXCEPTION_STACK_TRACE
 
 config EXCEPTION_STACK_TRACE_MAX_FRAMES
 	int "Configures the depth of stack trace"
+	default ARCH_STACKWALK_MAX_FRAMES
```

Comment:
I think this also needs `depends on ARCH_HAS_STACKWALK` as `ARCH_STACKWALK_MAX_FRAMES` is not always available.

Reply:
Good catch, fixed, thanks